gwsumm/tabs/gracedb.py

```diff
@@ -36,20 +37,22 @@
 __author__ = 'Duncan Macleod <duncan.macleod@ligo.org>'
 __all__ = ['GraceDbTab']
 
-LABELS = {
-    'ADVOK': 'success',
-    'ADVNO': 'danger',
-    'H1OPS': 'info',
-    'H1OK': 'success',
-    'H1NO': 'danger',
-    'L1OPS': 'info',
-    'L1OK': 'success',
-    'L1NO': 'danger',
-    'DQV': 'danger',
-    'INJ': 'warning',
+LABELS = OrderedDict()
+LABELS["danger"] = {
+    "ADVNO",
+    "DQV",
+    "H1NO",
+    "L1NO",
+    "V1NO",
+}
+LABELS["warning"] = {
+    "INJ",
+}
+LABELS["success"] = {
     'EM_READY': 'success',
+    'PASTRO_READY': 'success',
```

Comment:
@duncanmmacleod, should this entry to the `LABELS` dict also be a set, or did you intentionally leave it a dictionary?